docs/README.md

```diff
@@ -261,6 +261,9 @@ The valid values for this option are:
    * When set to **`start`**:
       * If there is information in the “state file”, use the last recorded state. 
       * If the file is not present in the “state file”, start at the beginning. 
+   * When set to **`tail`**:
+      * Will read .log files starting at the beginning if created after the agent has started. This can result in large .log files being processed if copied into log directory. A WARN log will be printed if this happens.  
+      
```

Comment:
what is the default? is it fixed in default case?

Reply:
the default case will still behave the same (none) 

this was the change that was discussed to address the bug.